src/pages/Forever/components/RedeemCode.tsx

```diff
@@ -103,24 +134,30 @@ export const RedeemCode: React.FC<RedeemCodeProps> = ({
                   type="text"
                   autoComplete="off"
                   placeholder=""
-                  touched={false}
-                  errors=""
+                  touched={touched.code}
+                  errors={errors.code ? textKeys[errors.code]() : ''}
                   variant="dark"
                 />
               </CodeField>
-            </Form>
-          )}
-        </Formik>
-      </Main>
+            </Main>
+            <SubmitButton
+              background={colorsV3.purple500}
+              foreground={colorsV3.gray900}
+              disabled={!values.code}
+            >
+              {textKeys.FOREVER_LANDINGPAGE_BTN_LABEL()}
+            </SubmitButton>
+          </RedeemForm>
+        )}
+      </Formik>
 
       <Footer>
-        <SubmitButton
-          background={colorsV3.purple500}
-          foreground={colorsV3.gray900}
-        >
-          {textKeys.FOREVER_LANDINGPAGE_BTN_LABEL()}
-        </SubmitButton>
+        <Info
+          dangerouslySetInnerHTML={{
+            __html: textKeys.FOREVER_LANDINGPAGE_INFO_TEXT(),
+          }}
+        />
```

Comment:
I think rather we could use [`{ MarkdownTranslation } from '@hedviginsurance/textkeyfy'`](https://github.com/HedvigInsurance/web-onboarding/blob/60642f8d28ec70aafa744956dc26ce90d6e5fb5b/src/pages/OfferNew/Checkout/Sign.tsx#L138-L151)

Reply:
Ah nice, that's better!